source/css/sass/patterns/molecules/teaser.scss

```
@@ -0,0 +1,30 @@
+@import "../../mixins/spacing";
+@import "../../mixins/typography";
+
+.teaser-list {
+
+  .teaser__header {
+    @include block-spacing($start: 0);
+  }
+
+  .teaser-list__list {
+    @include padding(0, inline-start);
+  }
+
+  .teaser-list__item {
```

No need to nest these selectors.

---

Are you allowing this sort of syntax in the pattern library:
```suggestion
  &__item {
```
for BEM classes?

---

@davidcmoulton and I have talked about this in the past, and we don't use it as it's harder to read and discover selectors (eg can't search the codebase for one).